azure-pipelines.yml

```diff
@@ -114,7 +114,12 @@ jobs:
     - bash: |
         set -e
         sudo apt-get update
-        sudo apt-get -y -q --no-install-recommends install zlib1g-dev libncursesw5-dev libgeoip-dev nettle-dev libgmp-dev libcurl4-gnutls-dev libsdl2-dev libogg-dev libvorbis-dev libopusfile-dev libwebp-dev libjpeg8-dev libpng-dev libfreetype6-dev libglew-dev libopenal-dev liblua5.2-dev ninja-build $(EXTRA_PACKAGES)
+        sudo apt-get -q install -y gdebi-core
+        wget http://mirrors.kernel.org/ubuntu/pool/universe/g/glew/libglew2.2_2.2.0-4_amd64.deb
```

Comment:
That is an option, yeah.

Reply:
It seems better to me to not upgrade, so that we don't add something requiring 2.2 without noticing it next time.

Reply:
I suppose it depends on if we expect the more up-to-date GLEW version.

Reply:
The 2.2.0 release is now 3.5 years old.

Reply:
We usually try to support building on Linux that old. See for example https://github.com/DaemonEngine/Daemon/issues/475

Reply:
Ah, yep, that makes sense. Ideally we'd build with one of Clang or GCC with newer version since they sometimes catch errors that the Windows build doesn't, but given that this is so far the only feature requiring 2.2 it should be fine.

Reply:
Removed the commit that changed azure pipelines now.

Reply:
We build the releases on an old Debian with an old libc, but in the release build script we build latest GLEW ourselve, so we always build and “old” binary (running on old distros) but with latest GLEW. As long as at least one CI target tests the build against a recent GLEW it's OK.

Reply:
I see. I consider this resolved then.